hubmap/frontend/cypress/integration/stub_globus_auth.js

```diff
@@ -0,0 +1,40 @@
+describe('The Login Page', function () {
+    before(() => {
+      Cypress.Cookies.debug(true)
+      cy.server();
+      const api = 'http://localhost:8000/';
+      cy.route(`${api}/api/?format=json`, 'fixture:base.json');
+      cy.route(`${api}/api/colors/?format=json`, 'fixture:colors.json');
+      cy.route(`${api}/api/genes/?format=json`, 'fixture:genes.json');
+    });
+    beforeEach(()=> {
+        cy.server()
+        .stubLoginUser()
+    })
+   
+    afterEach(()=> {
+        cy.server()
+        .stubLogoutUser();
+      const api = 'http://localhost:8000/';
+      cy.route(`${api}/api/?format=json`, 'fixture:base.json');
+      cy.route(`${api}/api/colors/?format=json`, 'fixture:colors.json');
+      cy.route(`${api}/api/genes/?format=json`, 'fixture:genes.json');
+      cy.visit('/');
+      cy.contains('Login');
+      Cypress.Cookies.debug(false);
+    })
+    it('sets auth cookie when logging in via form submission', function () {
+        cy.visit('/');
+
+      cy.get('#loggedin-menu').should('exist');
+  
+      // auth cookie should be present
+      cy.getCookie('email').should('exist');
+  
+      // UI should reflect this user being logged in
+      cy.get('#loggedinemail').should('contain', 'test@gmail.com').should('exist')
+      cy.get('#loggedin-menu').children().get('#logout-menuitem').should('exist')
```

Comment:
I would avoid making the tests too dependent on precise paths... Thinking about writing tests based on what the user sees on the screen instead. Like, there should be a "Logout" on the screen, and clicking should do the right thing.

Reply:
@mccalluc Well, honestly, this is url logout, so as your issue #98 , we were to test globus auth, we should test complete login and logout. The link exists and is in dropdown menu. Can you please test login/logout manually https://data.dev.hubmapconsortium.org/  and let me know why you are not able to find logout ? Please let me know about this, will be glad to fix any bugs if you don't find logout.